terraform/README.md

```diff
@@ -35,12 +39,12 @@ resource "juju_integration" "nrf-db" {
 
```

Comment:
Can't create a suggestion, but we should replace the reference to `var.model_name` with `juju_model.my_model.name`

Reply:
Fixed